src/partials/80_section-Feedback.html

```diff
@@ -0,0 +1,36 @@
+<div class="section--feedback">
+  <div class="container">
+    <div class="panel-bar">
+      <div class="row no-gutters align-items-end">
+        <div class="col-auto heading"><h3>Client Feedback</h3></div>
+        <div class="col menu">
+          <ul>
```

Comment:
Czemu ma służyć ta pusta lista?

Reply:
Zgodnie z daily --> http://prntscr.com/o0466h
ma to zostać usunięte?